Sources/SwiftKueryORM/Database.swift

```diff
@@ -0,0 +1,43 @@
+/**
+ Copyright IBM Corporation 2018
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import SwiftKuery
+import KituraContracts
+
+public enum ConnectionStrategy {
+  case global(Connection)
+  case pool(ConnectionPool)
+  case generator(() -> Connection?)
+}
+
+public typealias ConnectionPoolOptions = SwiftKuery.ConnectionPoolOptions
+public class Database {
```

Comment:
I think we need to revise the way Database is designed. Take a look at https://gist.github.com/tunniclm/1a73eee526c89f2200c81d7137c77e6b for some possible revisions.

Reply:
done